File: tests/benchmarks/cases/prime_trial_division/dotnet/benchmark_case.cs
namespace BenchmarkRunner;

internal static class PrimeTrialDivisionCase
{
    public const string Name = "prime_trial_division";
    public const string PassBanner = "BENCH_PRIME_TRIAL_DIVISION_PASS";

    public static long Run(int scale) => BenchmarkSupport.PrimeTrialDivision(scale);
}

File: tests/benchmarks/cases/prime_trial_division/java/benchmark_case.java
final class PrimeTrialDivisionCase {
    static final String NAME = "prime_trial_division";
    static final String PASS_BANNER = "BENCH_PRIME_TRIAL_DIVISION_PASS";

    private PrimeTrialDivisionCase() {}

    static long run(int scale) {
        return BenchmarkSupport.primeTrialDivision(scale);
    }
}
